### DISCUSSION TITLE
Para el Bonus de despliegue (Montar la página en línea con dominio gratuito) que dominio utilizan compañeros explorers?

### DISCUSSION THREAD

**Discussion by geraodst117** [Launch-X-Latam/MisionFrontEnd#288] [Q&A]
No conozco ningun dominio gratuito por lo que seria interesante comentar cual y como los utilizan para apoyarnos explorers

**Comment by DiegoDominguez3132** [comment]
Hola, para montar la pagina puedes usar github pages aca te dejo un tutorial de como puedes hacerlo: https://www.youtube.com/watch?v=sLTNgxxSBR4&t=251s

**Comment by Jramo5** [comment]
Hola amigo, puedes registrar un dominio gratuito por 1 mes, 3 meses, 6 meses u 12 mese en ---> [Freenom.com](https://www.freenom.com/es/index.html?lang=es)

puedes subir tu sitio en un server gratuito como [infinityfree.net](https://infinityfree.net)
